tests/v2/integration/catalogv2/cluster_repo_test.go

```diff
@@ -529,6 +529,32 @@ func (c *ClusterRepoTestSuite) TestOCIRepo5() {
 	})
 }
 
+// TestOCIRepoMultipleChartRepos tests CREATE, UPDATE, and DELETE operations of OCI ClusterRepo with many chart repos
+func (c *ClusterRepoTestSuite) TestOCIRepoMultipleChartRepos() {
+	//start registry
+	ts, err := StartRegistry(c)
+	assert.NoError(c.T(), err)
+
+	defer ts.Close()
+
+	u, err := url.Parse(ts.URL)
+	require.NoError(c.T(), err)
+
+	//push testingchart helm chart
+	for i := 0; i < 300; i++ {
```

Comment:
Why 300 times ?

Reply:
It's just an arbitrary number that was sure to cause problems before. I can lower this amount, but since this is done locally the performance won't change that much

Reply:
okay.